synapse/storage/databases/main/events_worker.py

```diff
@@ -129,7 +145,7 @@ class _EventRow:
     json: str
     internal_metadata: str
     format_version: Optional[int]
-    room_version_id: Optional[int]
+    room_version_id: Optional[str]
```

Comment:
`room_version_id` comes from the `rooms` table, where it is a string instead of an int.

Reply:
This looks correct, matrix room versions are technically supposed to be strings, not ints